archivebox/cli/archivebox_list.py

```diff
@@ -46,6 +46,16 @@ def main(args: Optional[List[str]]=None, stdin: Optional[IO]=None, pwd: Optional
         action='store_true',
         help="Print the output in JSON format with all columns included.",
     )
+    group.add_argument(
+        '--html',
+        action='store_true',
+        help="Print the output in HTML format"
+    )
+    parser.add_argument(
+        '--index',
+        action='store_true',
+        help='Include the index additional structures'
```

Comment:
This might be worth clarifying somewhere, what exactly are the additional structures?

Reply:
Maybe these are confusing...what about `--header` or `--include-header`? That is more explicit, and works for both html and csv.

Reply:
I like `--headers` or `--with-headers`.

Reply:
https://github.com/pirate/ArchiveBox/pull/452/commits/ae94dbfd80b195ab74a0e0690a7d1e8de9665777